### PR TITLE
va-textarea: remove v1

### PR DESCRIPTION
**Pull request by powellkerry**
## Chromatic

https://3030-va-textarea-v1-removal--65a6e2ed2314f7b8f98609d8.chromatic.com

## Description
This PR removes the v1 option for textarea including associated styling and tests. 
Closes [3030](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3030)

## QA Checklist
- [ x] Component maintains 1:1 parity with design mocks
- [ x] Text is consistent with what's been provided in the mocks
- [ x] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ x] Tab order and focus state work as expected
- [ x] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ x] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ x] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
No visual differences for v3
Screenshot 2024-07-26 at 11 15 27 AM

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-textarea/va-textarea.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@
 @use 'usa-textarea/src/styles/usa-textarea';
 @use 'usa-character-count/src/styles/_usa-character-count';
 
+@import '../../mixins/accessibility.css';
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-textarea/va-textarea.scss:11]: I think we might be able to remove this import too. It imports `.sr-only` but we should probably be replacing that with the USWDS `.usa-sr-only` class now using this import:

```
@use 'uswds-helpers/src/styles/usa-sr-only';
```

File: packages/web-components/src/components/va-textarea/va-textarea.tsx [hunk 2]
```diff
@@ -210,143 +204,104 @@ export class VaTextarea {
...
-              value={value}
-              part="input-type-textarea"
-            />
-            {!charcount && maxlength && value?.length >= maxlength && (
-                <span class={messageClass} aria-live="polite">
-                {i18next.t('max-chars', { length: maxlength })}
+                )
+              }
+              {headerAriaDescribedby && (
+                <span id="header-message" class="sr-only">
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-textarea/va-textarea.tsx:257]: This should be using `.usa-sr-only` now.

File: packages/web-components/src/components/va-textarea/va-textarea.tsx [hunk 3]
```diff
@@ -210,143 +204,104 @@ export class VaTextarea {
...
+              {i18next.t('max-chars', { length: maxlength })}
+              </span>
+          )}
+          {charcount && maxlength && (
+            <span id="charcount-message" class={messageClass} aria-live="polite">
+              {getCharacterMessage(value, maxlength)}
+            </span>
           )}
           {messageAriaDescribedby && (
             <span id="input-message" class="sr-only dd-privacy-hidden">
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-textarea/va-textarea.tsx:299]: Also replace with `.usa-sr-only`

**Comment by powellkerry** [comment]
@jamigibbs Good catch, the sr-only changes should be fixed now.

**Review by jamigibbs — APPROVED** [review]
👏🏻

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**